org.overte.Overte.yml

```diff
@@ -0,0 +1,88 @@
+app-id: org.overte.Overte
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+# QtWebEngine is not part of the KDE runtime,
+# so we use this package, which is also maintained by KDE developers.
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-23.08'
+command: Overte
+separate-locales: false
+rename-icon: Overte
+finish-args:
+  - --share=network
+  - --share=ipc # For Discord integration
+  - --device=dri # Graphics driver
+  - --socket=pulseaudio
+  # Overte requires OpenGL ES 3.2. While most Mesa drivers don't have this fully implemented,
+  # we try anyway, since it works on some devices.
+  - --env=MESA_GLES_VERSION_OVERRIDE=3.2
+  # QtWebEngine is shipped by the baseapp and not by the Sdk, which makes it use a non-standard path.
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  # KDE seems to do some aggressive theming which breaks things.
+  # See issue: https://github.com/overte-org/overte/issues/167
+  - --env=XDG_CURRENT_DESKTOP=foo
+  # While we technically support Wayland, there are issues with mouse capture: https://github.com/overte-org/overte/issues/628
+  - --socket=x11
+  - --env=QT_QPA_PLATFORM=xcb
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: Overte
+    buildsystem: simple
+    cleanup:
+      - squashfs-root
```

Comment:
```suggestion
```

this does nothing.

Reply:
Does this not delete the `squashfs-root` directory created by extracting the AppImage?